ams/ams-server/src/main/java/com/netease/arctic/ams/server/model/TableOptimizeInfo.java

```diff
@@ -40,6 +40,8 @@ public TableOptimizeInfo(TableIdentifier tableIdentifier) {
   private double quota = 0.0;
   private double quotaOccupation = 0.0;
 
+  private String groupName = "default";
```

Comment:
How about change the value to `TableProperties.SELF_OPTIMIZING_GROUP_DEFAULT`?

Reply:
Ok, I will change. and this pr will be close and will be fixed in https://github.com/majin1102/arctic/pull/2.